### PR TITLE
update how to restore backup

### PR DESCRIPTION
**Pull request by katamaks**
updated text, added image

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Language review completed. Will be merged after technical review.

File: howtogeneral/mendixcloud/how-to-restore-a-backup.md [hunk 1]
```diff
@@ -11,43 +11,78 @@ category: "Mendix Cloud"
...
-3.  Go to the **Projects** tab.
-4.  Select the project from you want to download the backup.
-5.  Go to the **Deploy** tab.
-6.  Go to the **Backup** tab.
-7.  Select the backup you want to restore and click **Restore Backup**.
-8.  Select the environment the backup should be restored to and click **Restore backup**.
-9.  Click **Yes** to restore the backup to the selected environment.
+If the app is still running, you have to stop it by clicking **Stop Application**. Then click **Restore Backup** again.
+
+{% endmarkdown %}</div>
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/how-to-restore-a-backup.md]: is "endmarkdown" as intended? It is visible in the github preview

File: howtogeneral/mendixcloud/how-to-restore-a-backup.md [hunk 2]
```diff
@@ -11,43 +11,78 @@ category: "Mendix Cloud"
...
+
+There are two methods for restoring a backup for a licensed cloud node. The first method is to restore the backup online. The second method is to download the backup first and then upload it to the Developer Portal.
+
+#### 3.2.1 Method 1
 
-## 3 Restore A Backup
+To restore a backup that is stored in the Mendix Cloud, follow these steps:
 
-The process for restoring a backup from a Free App and a licensed cloud node are different. Refer to the chapter applicable to your situation for steps on how to restore a backup.
+1. Go to the [MEndix Developer Portal](http://home.mendix.com).
```
Comment (jtwaleson) [inline howtogeneral/mendixcloud/how-to-restore-a-backup.md]: MEndix -> Mendix

File: howtogeneral/mendixcloud/how-to-restore-a-backup.md (hunk 1 above)
Comment (katamaks) [inline howtogeneral/mendixcloud/how-to-restore-a-backup.md]: yes it's correct that way

**Pull request merged**